src/LeanplumInternal.ts

```diff
@@ -208,6 +216,8 @@ export default class LeanplumInternal {
           this._varCache.setVariantDebugInfo(getVarsResponse[Constants.KEYS.VARIANT_DEBUG_INFO])
         }
 
+        // TODO: trigger messagesReceived
```

Comment:
Shouldn't this be addressed before merging?

Reply:
Yep, added, thanks.